scripts/cangen/cangen/traffic_calculation.py

```diff
@@ -0,0 +1,61 @@
+class BusLoadCalculator:
```

Comment:
Put a header comment at the top of the file with your name and date. Also add a link to your can bus article. see the other cangen files for a header example